### PR TITLE
Cypress/E2E: Fix failed tests related to Global header

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Test report
https://automation-dashboard.vercel.app/cycles/9ed911cd-94d7-4aba-8ca7-c90608eec356

#### Screenshots
Some of the screenshots
Screen Shot 2021-09-10 at 2 28 48 PM
Screen Shot 2021-09-10 at 3 51 20 PM
Screen Shot 2021-09-10 at 4 03 50 PM
Screen Shot 2021-09-10 at 4 51 30 PM
Screen Shot 2021-09-10 at 4 57 54 PM
Screen Shot 2021-09-10 at 5 24 43 PM
Screen Shot 2021-09-10 at 5 27 33 PM
Screen Shot 2021-09-10 at 5 37 44 PM
Screen Shot 2021-09-10 at 5 50 09 PM
Screen Shot 2021-09-10 at 5 53 48 PM
Screen Shot 2021-09-10 at 6 10 54 PM
Screen Shot 2021-09-10 at 6 22 22 PM
Screen Shot 2021-09-10 at 6 23 34 PM
Screen Shot 2021-09-10 at 6 41 53 PM
Screen Shot 2021-09-10 at 6 44 57 PM
Screen Shot 2021-09-10 at 6 51 52 PM
Screen Shot 2021-09-10 at 6 58 20 PM
Screen Shot 2021-09-10 at 7 01 02 PM
Screen Shot 2021-09-10 at 7 06 08 PM
Screen Shot 2021-09-10 at 7 52 36 PM
Screen Shot 2021-09-10 at 7 55 17 PM
Screen Shot 2021-09-10 at 8 28 33 PM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/system_console/site_configuration/link_customization_not_cloud_spec.js [hunk 1]
```diff
@@ -104,67 +109,49 @@ describe('SupportSettings', () => {
         cy.get('#privacy_link').should('not.exist');
     });
 
-    it('MM-T1037 - Customization Custom Support Email', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/site_configuration/link_customization_not_cloud_spec.js:107]: Intentionally deleted. See https://mattermost.atlassian.net/browse/MM-38468.

File: e2e/cypress/support/ldap_commands.js [hunk 2]
```diff
@@ -32,91 +28,12 @@ Cypress.Commands.add('performLDAPLogin', (settings = {}, useEmail = false) => {
...
-                });
-            });
-
-            cy.findByText('Invite Members').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.HALF_SEC).then(() => {
-                cy.findByText('Invite More People').scrollIntoView().should('be.visible').then(() => {
-                    cy.findByText('Done').scrollIntoView().should('be.visible').click();
-                });
-            });
-        });
-    });
```
Comment (saturninoabril) [inline e2e/cypress/support/ldap_commands.js:119]: Commands here are no longer use anywhere.

File: e2e/cypress/support/ui_commands.js [hunk 3]
```diff
@@ -484,16 +472,6 @@ Cypress.Commands.add('minDisplaySettings', () => {
...
 
-// Reverts theme color changes to the default Mattermost theme
-Cypress.Commands.add('defaultTheme', (username) => {
-    cy.toAccountSettingsModal(username);
-    cy.get('#displayButton').click();
-    cy.get('#themeEdit').click();
-    cy.get('#standardThemes').click();
-    cy.get('.col-xs-6.col-sm-3.premade-themes').first().click();
-    cy.get('#saveSetting').click();
-});
```
Comment (saturninoabril) [inline e2e/cypress/support/ui_commands.js:495]: Commands no longer used.

**Review by nevyangelova — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]
Thanks @saturninoabril , sorry for the delay, I was trying to run in Pipeline but looks like Evn variable is not getting set properly, or I am missing something. 
I had run locally, all looks good ✅ 

image

image

image

image

image

image

image


image


image


image


image


image

image

**Comment by saturninoabril** [comment]
Dean, sorry but I hope it's fine with you to merge this now so I'm unblock in fixing/submitting another batch of test fixes for Global header. Feel free to comment so I could implement in the next PRs.

**Comment by saturninoabril** [comment]
Done cherry-pick into release-6.0

**Pull request merged**